src/cgame/cg_tutorial.cpp

```diff
@@ -441,55 +476,62 @@ static void CG_HumanText( char *text, playerState_t *ps )
 	if ( nextWeapon != WP_NONE )
 	{
 		Q_strcat( text, MAX_TUTORIAL_TEXT,
-		          va( _( "Press %s to use the %s\n" ),
+		          va( _( "Press %s to use the %s" ),
 		              CG_KeyNameForCommand( "weapnext" ),
 		              _( BG_Weapon( nextWeapon )->humanName ) ) );
+		Q_strcat( text, MAX_TUTORIAL_TEXT, "\n" );
 	}
 
 	if ( BG_InventoryContainsUpgrade( UP_JETPACK, ps->stats ) )
 	{
-		Q_strcat( text, MAX_TUTORIAL_TEXT, va( _( "Hold %s to use the %s\n" ),
+		Q_strcat( text, MAX_TUTORIAL_TEXT, va( _( "Hold %s to use the %s" ),
```

Comment:
We would get better translation if strings like this said `Hold %s to use the jetpack` rather than `Hold %s to use the %s`.

Reply:
Well, this hardcoding is only doable for two or three things, the jetpack, the medkit, and the armoury, probably no more. If we consider those names not common names but proper nouns (like if we name a branded product), I guess this would work in most languages. Languages that would be badly translated are the ones with declensions (like Latin) because those names would lack the proper declension (here the accusative one), but then only those 2 or 3 sentences among all the tutorial ones would be correct. I would take care of those 2 or 3 strings the day we want to make all strings from tutorial correct in such languages. We may even chose another solution, where declension languages would provide the accusative variation for the word and the function would pick the accusative variation if it exists. So to me it's too early to decide on that.

Reply:
I finally did what's suggested (hardcoding those names), as there is already some sentences doing that, for example:

```sh
_( "Find an Armoury, Drill or Reactor for more ammo." ) );
_( "Find an Armoury for more ammo." ) );
```

But I would not oppose to doing the absolute opposite and modifying those other sentences instead, so they rely on entity names programmatically.